react-native.config.js

```diff
@@ -15,6 +15,16 @@
  */
 'use strict';
 
+let supportsCodegenConfig = true;
+// try {
+//   const rnCliAndroidVersion = require('@react-native-community/cli-platform-android/package.json')
+//     .version;
+//   const [major] = rnCliAndroidVersion.split('.');
+//   supportsCodegenConfig = major >= 9;
+// } catch (e) {
+//   // ignore
+// }
```

Comment:
Should it be left like that?

Reply:
It was this way because the commented part was picking the rncli version from the root package instead of the fabric example.

I made a workaround for it in 1f41b69cb0318d9fd482d396d7a46aa9f2e94305, I added a flag that allows to enable the custom config and made a patch to the cli so that it runs with the added flag.